README.md

```diff
@@ -0,0 +1,39 @@
+# Open Source Programs (Backend)
+Open Source Programs (OSP) is an application that simplifies the processing and selection procedure of Open Source Programs of AnitaB.org Open Source or other third-party programs. This is the Backend repo for OSP.
+
+## Tech Stack
+- Django
+- Django REST Framework
+
+## Setup 
+To setup the project locally go through [this wiki page](https://github.com/anitab-org/open-source-programs-web/wiki/Fork,-Clone,-Remote-and-Pull-Request).
+Make sure you have [pip](https://pip.pypa.io/en/stable/installing/) and [virtualenv](https://pypi.org/project/virtualenv/) installed.
+
+1. Database Setup
+<br>
+Before starting with the project create a db in you local using PostgreSQL with the following details. Refer to `main/settings.py` if you have any confusion.
```

Comment:
I suggest you add a few SQL commands( eg : createdb etc)  for setting up PostgreSQL